docker-compose.yml

```diff
@@ -518,6 +519,12 @@ services:
         KAFKA_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM: none
       volumes:
         - ./tests/data/localhost.p12:/certs/localhost.p12:ro
+    pulsar:
+      image: apachepulsar/pulsar
+      command: bin/pulsar standalone
+      ports:
+        - "6650:6650"
+        - "8080:8080"
```

Comment:
It appears the tests are only accessing port 6650. If port 8080 is not needed for testing, it should probably be left out here.